examples/gno.land/r/faucet/faucet.gno

```diff
@@ -9,60 +9,88 @@ import (
 
 var (
 	// configurable by admin.
-	gAdminAddr       std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gControllerAddr  std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gPerTransferSend             = std.Coins{std.Coin{"ugnot", 1000000}}
-	gInPause                     = false
-	gMessage                     = "# Community Faucet.\n\n"
+	gAdminAddr   std.Address    = "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5"
+	gControllers [3]std.Address = [3]std.Address{} // limit it to 3
+
+	gInPause = false
+	gMessage = "# Community Faucet.\n\n"
 
 	// internal vars, for stats.
 	gTotalTransferred std.Coins
 	gTotalTransfers   = uint(0)
+
+	// per account request limit 350 gnot
+	limit std.Coins = std.Coins{std.Coin{"ugnot", 350000000}}
 )
 
-func Transfer(to std.Address) error {
+func Transfer(to std.Address, send int64) string {
 	if err := assertIsController(); err != nil {
-		return err
+		return err.Error()
 	}
 
 	if gInPause {
-		return errors.New("faucet in pause")
+		return errors.New("faucet in pause").Error()
 	}
 
-	send := gPerTransferSend
+	var sendCoins std.Coins
+
+	if send <= limit.AmountOf("ugnot") {
+
+		sendCoins = std.Coins{std.Coin{Denom: "ugnot", Amount: send}}
+
+	} else {
+
+		return errors.New("Per request limit " + limit.String() + " exceed").Error()
+	}
 
-	gTotalTransferred = gTotalTransferred.Add(send)
+	gTotalTransferred = gTotalTransferred.Add(sendCoins)
 	gTotalTransfers++
 
-	banker := std.GetBanker(std.BankerTypeOrigSend)
+	banker := std.GetBanker(std.BankerTypeRealmSend)
 	pkgaddr := std.GetOrigPkgAddr()
-	banker.SendCoins(pkgaddr, to, send)
-	return nil
+	banker.SendCoins(pkgaddr, to, sendCoins)
+	return ""
 }
+func GetPerTransferLimit() int64 {
 
+	return limit.AmountOf("ugnot")
+
+}
 func Render(path string) string {
-	banker := std.GetBanker(std.BankerTypeOrigSend)
+	banker := std.GetBanker(std.BankerTypeRealmSend)
 	balance := banker.GetCoins(std.GetOrigPkgAddr())
 
-	output := gMessage
+	output := path + gMessage
```

Comment:
```suggestion
	output := gMessage
```

nope?

Reply:
The output needs to include the path. Otherwise, it will not be rendered from the webpage.